src/pages/Menu/Components/Card/Card.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Card.scss';
+
+const Card = props => {
+  const { product } = props;
```

Comment:
```suggestion
const Card = ({ product })=> {
```
매개변수 자리에서 바로 구조분해 할당 가능합니다!

Reply:
넵 수정하였습니다 !